addons/pos_self_order/static/src/overrides/components/order_receipt/order_receipt.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="pos_self_order.OrderReceipt" t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('receipt-change')]" position="attributes">
+            <attribute name="class">pos-receipt-amount receipt-change</attribute>
```

Comment:
if you really have to do this, it would be better to simply remove the `mt-2` directly in pos

Reply:
@vlst-odoo According to specs, changes in receipt were only focused on self-order/kiosk.
Thanks for the suggestion, It will be more accurate to changes in pos.